gallery-time/src/Galleries/Galleries.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Galleries.css';
+import { galleriesSearch } from '../apiCalls.js';
+import { denverGalleries } from '../MockData/MockData.js';
+import  GalleryCard from '../GalleryCard/GalleryCard.js';
+
+const Galleries = ({ geoLocation }) => {
+  const [galleries, setGalleries] = useState();
```

Comment:
instead of galleries have galleryObjects 
instead of galleriesDisplay galleryComponents